### PR TITLE
fix(dist): bin package lack execute permission

### PR DESCRIPTION
**Pull request by z7658329**

### CODE REVIEW

File: hugegraph-loader/pom.xml [hunk 1]
```diff
@@ -650,9 +650,11 @@
                         <configuration>
                             <target>
                                 <tar destfile="${project.basedir}/${final.name}.tar.gz"
```
Comment (coderzc) [inline hugegraph-loader/pom.xml:652]:
```suggestion
                                <tar destfile="${project.basedir}/target/${final.name}.tar.gz"
```

File: hugegraph-tools/pom.xml [hunk 2]
```diff
@@ -235,9 +235,11 @@
                         <configuration>
                             <target>
                                 <tar destfile="${project.basedir}/${final.name}.tar.gz"
```
Comment (coderzc) [inline hugegraph-tools/pom.xml:237]:
```suggestion
                                <tar destfile="${project.basedir}/target/${final.name}.tar.gz"
```

File: hugegraph-loader/pom.xml [hunk 3]
```diff
@@ -650,9 +650,11 @@
...
-                                     basedir="${project.basedir}"
-                                     includes="${final.name}/**"
-                                     compression="gzip"/>
+                                     compression="gzip">
+                                    <tarfileset dir="${project.basedir}" filemode="755">
```
Comment (coderzc) [inline hugegraph-loader/pom.xml:654]: why is `${project.basedir}`?

Comment (z7658329) [reply hugegraph-loader/pom.xml:654]: because plugins will generate sources in this path

File: hugegraph-loader/pom.xml (hunk 1 above)
Comment (z7658329) [reply hugegraph-loader/pom.xml:652]: the generate binary's dest path should not be project's target dir ? @coderzc @imbajin

Comment (imbajin) [reply hugegraph-loader/pom.xml:652]: > the generate binary's dest path should not be project's target dir ? @coderzc @imbajin

it should under the target, but be careful to check the relative influence like in computer's CI or some script

**Review by coderzc — APPROVED** [review]

File: hugegraph-tools/pom.xml [hunk 4]
```diff
@@ -187,11 +187,6 @@
     </dependencies>
 
     <build>
-        <directory>output</directory>
-        <sourceDirectory>src/main/java</sourceDirectory>
-        <!--<testSourceDirectory>src/test/java</testSourceDirectory>-->
-        <outputDirectory>output/classes</outputDirectory>
-        <!--<testOutputDirectory>output/test-classes</testOutputDirectory>-->
```
Comment (imbajin) [inline hugegraph-tools/pom.xml:194]: @javeme ensure is it useless now?

**Review by imbajin — APPROVED** [review]

**Pull request merged**